app/serializers/v1/coupon_serializer.rb

```diff
@@ -14,10 +14,15 @@ def serialize
         frequency: model.frequency,
         frequency_duration: model.frequency_duration,
         reusable: model.reusable,
+        limited_plans: model.limited_plans,
         created_at: model.created_at.iso8601,
         expiration: model.expiration,
         expiration_at: model.expiration_at&.iso8601,
       }.merge(legacy_values)
+
+      payload = payload.merge(plans) if include?(:plans)
```

Comment:
More a question, do we need the full plan payload? What about only returning the plan code or id list? Or maybe make these include optional?

Reply:
Probably not needed to return everything:) returning `plan_codes` now